xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_version.xsd

```diff
@@ -201,6 +201,21 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Key issuing scheme under which facility is accessible.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="HandWashing" type="xsd:boolean" default="true" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether the toilets have a sink allowing hand washing with soap.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="DrinkingWater" type="xsd:boolean" default="true" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether the tap inside the toilet supplies drinking water.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="ToiletsPosition" type="ToiletsPositionEnumeration" minOccurs="0">
```

Comment:
@nick-knowles  pointed that Position is misleading in English ... Type appears to be better